front/app/containers/Admin/pagesAndMenu/containers/CustomPages/Edit/HeroBanner/index.tsx

```diff
@@ -32,14 +42,18 @@ export type CustomPageBannerSettingKeyType = Extract<
   | 'banner_cta_button_type'
 >;
 
+const StyledStatusLabel = styled(StatusLabel)`
+  height: 20px;
+`;
```

Comment:
The font-weight is higher in the design (700). Could you add HeightBoxProps to the component in the library (preferred) or wrap with Box instead?

(Even easier, it also looks fine without specifying the height imo. Would be different than the design, but it doesn't look bad.)

Reply:
Refactoring this component to introduce it for top/bottom info sections, will change